python/lsst/obs/subaru/gen3/hsc/rawFormatter.py

```diff
@@ -39,25 +39,14 @@ class HyperSuprimeCamRawFormatter(FitsRawFormatterBase):
     FLIP_LR = True
     FLIP_TB = False
     translatorClass = HscTranslator
+    filterDefinitions = HSC_FILTER_DEFINITIONS
 
     def makeWcs(self):
         wcs = makeSkyWcs(self.metadata, strip=True)
         dimensions = bboxFromMetadata(self.metadata).getDimensions()
         center = Point2D(dimensions/2.0)
         return makeFlippedWcs(wcs, self.FLIP_LR, self.FLIP_TB, center)
 
-    def makeFilter(self):
-        # For historical reasons we need to return a short, lowercase filter
-        # name that is neither a physical_filter nor an abstract_filter in Gen3
```

Comment:
This comment is quite confusing because it seems it always used physical_filter for lookup.

Reply:
Yes, this is very confusing to me too. I think I've identified the issue that @TallJimbo was dealing with when he wrote this. We should talk about it today to decide on an approach.

Reply:
The new `FilterDefinition` system should be much more explicit about the various names, so I think we've cleared this up.